lib/screens/signInPage/sign_in.dart

```diff
@@ -40,10 +40,14 @@ class SignInPage extends StatelessWidget {
                 SignInButton(
                   onPressed: () => _handleSignIn(),
                 ),
-                RaisedButton(
-                  child: Text("Sign out"),
-                  onPressed: _authService.signOut,
-                ),
+                WillPopScope(
```

Comment:
Why this??

Reply:
If I press the backbutton you get back in the app
And use the app without being logged in